ddclient.in

```diff
@@ -7644,6 +7653,69 @@ Example ${program}.conf file entries:
 EoEXAMPLE
 }
 
+######################################################################
+## nic_ionos_examples
+######################################################################
+sub nic_ionos_examples {
+    my $self = shift;
+    return <<EoEXAMPLE;
+o 'ionos'
+
+The 'ionos' protocol is used by DNS service offered by ionos.com.
+
+Configuration variables applicable to the 'ionos' protocol are:
+  protocol=ionos                ##
+  password=dyndns-update-key    ## the key created to update the host below
+  fully.qualified.host          ## the host registered with the service.
+
+Example ${program}.conf file entries:
+  ## single host update
+  protocol=ionos,
+  password=dyndns-update-key
+  myhost.com
+
+Getting the DynDNS key (only available via Ionos API):
+
+  1. Create an API key
+  2. Enable DynDNS for your host using the /dns/v1/dyndns API
+     (documentation @ https://developer.hosting.ionos.es/docs/dns)
```

Comment:
```suggestion
     (documentation @ https://developer.hosting.ionos.com/docs/dns)
```

Reply:
Good catch. Not sure why I landed in the .es site.